themes/prologue/prologue.rst

```diff
@@ -46,17 +44,15 @@ Coding standard and guidelines
 General code guideline
 ~~~~~~~~~~~~~~~~~~~~~~
 
-Intend with spaces for every language (PHP, HTML, CSS, etc.): 4 spaces for PHP files, 2 spaces for all other filetypes.
+Indented with spaces for every language (PHP, HTML, CSS, etc.): 4 spaces for PHP files, 2 spaces for all other filetypes.
```

Comment:
Yep, there was an intend/indent typo, thanks!
But "Indent" here is a suggestion, so an imperative verb. "Indented" just doesn't work in that context.